### PR TITLE
New major: v5

### PR DESCRIPTION
**Pull request by fanatid**
Coin name changed to symbol. Right now SLIP have coin names like: `Global Currency Reserve (GCRcoin)`, `Virtual Cash the new version of Vpncoin`. Symbol much easier.

Constants updated.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,185 +1,232 @@
...
+  "DCR": 0x8000002a,
+  "DFC": 0x80000539,
+  "DGB": 0x80000014,
+  "DGC": 0x80000012,
+  "DLC": 0x80000066,
+  "DMD": 0x80000098,
+  "DNR": 0x80000074,
+  "DOGE": 0x80000003,
+  "DOPE": 0x80000035,
+  "DSH": 0x80000005,
```
Comment (dooglio) [inline index.js:69]: Why not `DASH`? Why this odd abbreviation?

Comment (jprichardson) [reply index.js:69]: It should be `DASH`, this is incorrect.

Comment (jprichardson) [reply index.js:69]: Ah, but it's incorrect on the list itself: https://github.com/satoshilabs/slips/blob/master/slip-0044.md

Comment (dooglio) [reply index.js:69]: Yeah, so we can come back to this when they change it in the official repo. Thanks for checking, @jprichardson!

**Pull request merged**